README.md

```diff
@@ -1,39 +1,37 @@
 
- * https://eurorust.eu
+- problems with inter process communication - too many copies
+- zero copy is the solution
+- goal, embedded safety certified code
+- link to roadmap, contribution, code of conduct, faq, maintainer
 
-## Famous C/C++ bugs
+## Examples
 
- * https://www.heise.de/security/alerts/
+### Getting Started
 
-### Examples
+### Publish Subscribe
 
- * https://www.heise.de/news/Schwachstelle-im-Linux-Kernel-ermoeglicht-Codeschmuggel-im-WLAN-7309762.html
+### Events
 
-## Why you should write in Rust instead of C++
+## Supported Platforms
 
- * https://media.defense.gov/2022/Nov/10/2003112742/-1/-1/0/CSI_SOFTWARE_MEMORY_SAFETY.PDF
+### Target Platforms
 
-## Projects which uses rust for safety/security reasons
+### Developer Platforms
 
- * https://aws.amazon.com/blogs/opensource/sustainability-with-rust/
- * https://www.dev-insider.de/rust-fuer-mehr-sicherheit-in-automotive-software-a-38d5d780a303c722559c31f305c4d5cb/
-    * https://www.elektrobit.com/newsroom/blackberry-and-elektrobit-bolster-automotive-safety-roadmap/
- * https://www.golem.de/news/kata-os-und-project-sparrow-google-legt-beweisbar-sicheres-betriebssystem-offen-2210-168988.html
-    * https://sel4.systems
-    * https://dl.acm.org/doi/10.1145/1743546.1743574
- * https://www.heise.de/news/Programmiersprache-Facebook-gefaellt-die-Rust-Foundation-und-tritt-ihr-bei-6032634.html
- * https://www.heise.de/developer/meldung/Kubernetes-mit-Rust-Microsoft-setzt-fuer-Cloud-Entwicklung-auf-Rust-4714513.html
- * https://blog.anp.lol/rust/2016/06/11/baby-steps-porting-musl-to-rust/
- * https://github.com/uutils/coreutils
+## Thanks To All Contributors
 
-## Links why you should write safety critical software in rust
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
 
- * https://www.heise.de/hintergrund/Entwicklung-Warum-Rust-die-Antwort-auf-miese-Software-und-Programmierfehler-ist-4879795.html
-   * https://www.heise.de/select/ix/2021/5/2104613402044260988
- * https://blog.torproject.org/announcing-arti/
- * https://jamey.thesharps.us/2017/01/03/which-projects-should-convert-to-rust/
- * https://serokell.io/blog/rust-companies
- * https://www.zdnet.com/article/programming-languages-its-time-to-stop-using-c-and-c-for-new-projects-says-microsoft-azure-cto/
-   * https://www.theregister.com/2022/09/28/is_it_time_to_retire_c/
- * https://medium.com/volvo-cars-engineering/why-volvo-thinks-you-should-have-rust-in-your-car-4320bd639e09
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/elboberido"><img src="https://avatars.githubusercontent.com/u/56729607" width="100px;" alt="Mathias Kraus"/><br /><sub><b>Mathias Kraus</b></sub></a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/elfenpiff"><img src="https://avatars.githubusercontent.com/u/56729169" width="100px;" alt="Christian Eltzschig"/><br /><sub><b>Christian Eltzschig</b></sub></a></td>
```

Comment:
What do you think about adding the github name, e.g. `Mathias Kraus aka elBoberido` or `Mathias »elBoberido« Kraus` or `elBoberido - Mathias Kraus`?

Reply:
I will use the second version, then I have the feeling that we sound like mafia gangsters ;)